src/main/features/response/views/more-time-request.njk

```diff
@@ -36,7 +35,8 @@
                 bold: true
               }
             ],
-            form = form
+            form = form,
+            legendHidden = true
```

Comment:
is the legend duplicating the label?

Reply:
have updated to make the label match the title

Reply:
FYI for future when we have similar cases:
> Ed Horsford [10 hours ago]
We now recommend nesting the h1 within the legend, so you don’t get anything repeated and everyone is happy.